owsi-core/owsi-core-components/owsi-core-component-jpa-more/pom.xml

```diff
@@ -62,6 +62,13 @@
 			<artifactId>imageinfo</artifactId>
 		</dependency>
 		
+		<!-- Diff -->
+		
+		<dependency>
```

Comment:
I still wonder if this wouldn't be better to add all this diff stuff to a separate maven module... Of course, we would have to keep some interfaces in here still (or in owsi-core-components-commons), but at least we'd be rid of this dependency for the 90% of our projects that do not use java-object-diff.